pysilcam/silcamgui/interactive_summary.py

```diff
@@ -544,22 +595,24 @@ def update_plot(self, save=False):
         string += '\n VC oil [uL/L]: {:0.0f}'.format(psd_vc_oil)
         string += '\n VC gas [uL/L]: {:0.0f}'.format(psd_vc_gas)
         string += '\n\n Num images: {:0.0f}'.format(psd_nims)
-        string += '\n\n Start: ' + str(pd.to_datetime(psd_start[0]))
-        string += '\n End: ' + str(pd.to_datetime(psd_end[0]))
-        string += '\n Window [sec.] {:0.3f}:'.format(pd.to_timedelta(psd_end[0]-psd_start[0]).total_seconds())
+        string += '\n\n Start: ' + str(psd_start)
+        string += '\n End: ' + str(psd_end)
+        string += '\n Window [sec.] {:0.3f}:'.format(pd.to_timedelta(psd_end-psd_start).total_seconds())
         string += '\n\n mid-time: ' + str(pd.to_datetime(self.mid_time))
 
-        plt.title(string, verticalalignment='top', horizontalalignment='right', loc='right')
+        self.axistext.clear()
+        self.axistext.text(1, 1, string, va='top', ha='right', transform=self.axistext.transAxes)
+        self.axistext.axis('off')
 
-        plt.sca(self.axisconstant)
+        #plt.sca(self.axisconstant)
```

Comment:
E265: block comment should start with '# '  <!-- Automatically posted by Lintly -->